src/main/java/io/cdap/plugin/gcp/bigquery/source/PartitionedBigQueryInputFormat.java

```diff
@@ -131,7 +131,7 @@ private String generateQuery(String partitionFromDate, String partitionToDate, S
       return null;
     }
     String queryTemplate = "select * from %s where %s";
-    com.google.cloud.bigquery.Table sourceTable = BigQueryUtil.getBigQueryTable(project, datasetProject, dataset, table,
+    com.google.cloud.bigquery.Table sourceTable = BigQueryUtil.getBigQueryTable(datasetProject, dataset, table,
```

Comment:
I believe the only other place this `getBigQueryTable` is still used is in  `io.cdap.plugin.gcp.bigquery.sink.BigQuerySink#configureTable`. Should it also use the one without two projects?

Reply:
It's also changed in the PR.